puffin_egui/src/flamegraph.rs

```diff
@@ -761,15 +761,20 @@ fn paint_scope(
             let Some(scope_details) = info.scope_collection.fetch_by_id(&scope.id) else {
                 return Ok(PaintResult::Culled);
             };
-            egui::show_tooltip_at_pointer(&info.ctx, Id::new("puffin_profiler_tooltip"), |ui| {
-                paint_scope_details(ui, scope.id, scope.record.data, scope_details);
-                add_space(ui);
-                ui.monospace(format!(
-                    "duration: {:7.3} ms",
-                    to_ms(scope.record.duration_ns)
-                ));
-                ui.monospace(format!("children: {num_children:3}"));
-            });
+            egui::show_tooltip_at_pointer(
+                &info.ctx,
+                LayerId::background(),
```

Comment:
This is the layer of the parent `ui`. Using `LayerId::background()` works if the flamegraph is in a background panel, but not if it is in a floating `Window`.

Instead, we should pass down `ui.layer_id()` to `paint_scope` so we can use it here

Reply:
Ah nice, take a look at that